web/recipes/sous_media/config/taxonomy.vocabulary.media_tags.yml

```diff
@@ -0,0 +1,8 @@
+uuid: a68a32a0-30af-42b3-9e45-bf0c2f70520e
```

Comment:
@laura-johnson should this file be added to the `sous_media` in `/assets/scaffold/recipes/sous_media` ?